src/app/docs/summary-action-bar/summary-action-bar-docs.component.html

```diff
@@ -0,0 +1,11 @@
+<sky-docs-demo-page
+  moduleName="SummaryActionBarModule"
+  moduleSourceCodePath="src/app/public/modules/summary-action-bar/"
+  pageTitle="Summary action bar"
+>
+
+<sky-docs-demo-page-summary>
+  The summary action bar module provides a docked container for actions and summary information on forms or multiselect lists.
+</sky-docs-demo-page-summary>
+
```

Comment:
We have some UX guidelines in place at https://developer.blackbaud.com/skyux/components/summary-actionbar#ux-guidelines. Do we have an issue to port that content over? If not, I think we should either create a separate issue or port that content over in this PR.